api_tests/base/test_filters.py

```diff
@@ -467,6 +474,52 @@ def test_filter_queryset_handles_multiple_fields(self):
             )]
         assert_equal(actual, [40, 30, 10, 20])
 
+    def test_sort_by_serializer_field(self):
+        user = AuthUserFactory()
+        NodeFactory(creator=user)
+        NodeFactory(creator=user)
+
+        # Ensuring that sorting by the serializer field returns the same result
+        # as using the source field
+        res_created = self.app.get('/{}nodes/?sort=created'.format(API_BASE), auth=user.auth)
+        res_date_created = self.app.get('/{}nodes/?sort=date_created'.format(API_BASE), auth=user.auth)
+        assert res_created.status_code == 200
+        assert res_created.json['data'] == res_date_created.json['data']
+        assert res_created.json['data'][0]['id'] == res_date_created.json['data'][0]['id']
+        assert res_created.json['data'][1]['id'] == res_date_created.json['data'][1]['id']
+
+        # Testing both are capable of using the inverse sort sign '-'
+        res_created = self.app.get('/{}nodes/?sort=-created'.format(API_BASE), auth=user.auth)
+        res_date_created = self.app.get('/{}nodes/?sort=-date_created'.format(API_BASE), auth=user.auth)
```

Comment:
I'd DRY up the base nodes URL either with a method where you pass in the sort field, and returns the entire URL, or adding the sort param to the base URL (+)